src/core/Table/filters/FilterButtonBar.test.tsx

```diff
@@ -75,3 +75,17 @@ it('should call callbacks on clicks', () => {
   screen.getByText('Clear').click();
   expect(clearFilter).toHaveBeenCalled();
 });
+
+it('should consume the click event and stop its propagation', () => {
```

Comment:
This test is failing now.

Reply:
fixed it, had to move the logic to basefilter as input box click was also toggling the sort